Utilize multiple address lines, not just the first line

gulshan-raj opened the pull request:
Send and receive multiple address lines

Commit:
Multi-Address Line issue Resolved

jsullivanlive commented:
Hard to read the diff with all the whitespace changes checked in, so here's a link to the whitespace ignored version:  https://github.com/cloudanswers/salesforce-freshbooks-sync/pull/1/files?w=0

Has this been tested on an address that has only one line?

It looks like the second line is added without a line break, so if you have a 2 line address in freshbooks it will end up being a 1 line address with both lines together on that line in salesforce.

@Avinava we should probably consider a git hook to do code formatting pre commit for pages and classes

Commit:
Multi-Address Line issue Resolved with newline 

Multi-Address Line issue Resolved with newline between two address lines coming from Freshbook

Commit:
MultiLine Address: Changed from \r\n to \n
